extension/src/markdown/YamlItem.ts

```diff
@@ -43,4 +43,26 @@ export class YamlItem {
     });
     return result;
   }
+
+  public static arrayToMarkdown(items: YamlItem[]): string {
+    let result = "";
+    items.forEach((item) => {
+      result += item.toMarkdown();
+    });
+    return result;
+  }
+  public toMarkdown(level = 0): string {
+    const indentation = "".padEnd(level + 2, "#");
+    let result = "";
+
+    result += `${indentation} [${this.name}](${
+      this.href.endsWith(".md") ? this.href : this.topicHref
+    })\n`;
+    if (this.items !== undefined) {
```

Comment:
Just checking for truthy seem like it would suffice :)

```suggestion
    if (this.items) {
```

Reply:
This was just a copy&paste from some code where you replaced isNullOrUndefined with this... 😂

I'll fix both now :)